client/layout/controller.js

```diff
@@ -32,13 +32,11 @@ const getPages = () => {
 			container: DevDocs,
 			path: '/devdocs',
 			wpOpenMenu: 'toplevel_page_woocommerce',
```

Comment:
I'm wondering if we should rethink the logic behind `wpOpenMenu` as well.

Reply:
`wpOpenMenu` doesn't seem as bad. It makes sense for a page to indicate which section it is on. Maybe not ideal to have to specify an ID that's auto-generated by WP-Admin, but not terrible.

Reply:
Ya, certainly not ideal.  I was thinking we could just find and add the open classes to the parent menu item when an item is clicked instead of specifying `wpOpenMenu`.

I'm wondering if there's a use case for using a different selector in `wpOpenMenu` other than the item's parent.  If so, we can leave it as is.